package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protocol",
-  "version": "2.3.1",
+  "version": "2.3.2",
```

Comment:
There are more things to bump than the version number here. I'd be happy to walk you through the release process, as it's not as simple as some npm packages.